plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -2,6 +2,7 @@ FFmpegOutput="FFmpeg Output"
 FFmpegAAC="FFmpeg Default AAC Encoder"
 FFmpegOpus="FFmpeg Opus Encoder"
 FFmpegOpts="FFmpeg Options"
+FFmpegOpts.ToolTip="Use to specify custom FFmpeg options. For example, \"level=5.2 profile=main\""
```

Comment:
These already exist as FFmpeg.Options and FFmpeg.Options.Tooltip in line 70.
Though this one here is shorter and gets to the point.

Reply:
I'd recommend the key to be `FFmpegOpts.ToolTip.Encoder` to indicate that this isn't universal, since the source has a different tooltip for custom options (#6519 brings the naming in line with my recommendation here (`FFmpegOpts.ToolTip.Source`)).